### PR TITLE
Add AnnaCurly's file

### PR DESCRIPTION
**Pull request by anna-kondratiuk**
created a branch, created a file and made a commit, and opened a pull request

### CODE REVIEW

File: _posts/0000-01-02-AnnaCurly.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (AnnaCurly) [inline _posts/0000-01-02-AnnaCurly.md:5]: "Welcome too , thank you for teching me"

Comment (AnnaCurly) [inline _posts/0000-01-02-AnnaCurly.md:5]: "this is my change"

Comment (AnnaCurly) [reply _posts/0000-01-02-AnnaCurly.md:5]: "so this is closed?"

**Review by anna-kondratiuk — COMMENTED** [review]
ok

**Review by anna-kondratiuk — COMMENTED** [review]
ok

**Review by anna-kondratiuk — COMMENTED** [review]
submit??

File: _posts/0000-01-02-AnnaCurly.md (hunk 1 above)
Comment (AnnaCurly) [inline _posts/0000-01-02-AnnaCurly.md:5]: "change this line here"

Comment (AnnaCurly) [reply _posts/0000-01-02-AnnaCurly.md:5]: "srart review"

**Review by anna-kondratiuk — COMMENTED** [review]
"this is ok"

**Comment by anna-kondratiuk** [comment]
this is reviewed

File: _posts/0000-01-02-AnnaCurly.md (hunk 1 above)
Comment (anna-kondratiuk) [inline _posts/0000-01-02-AnnaCurly.md:5]: *this is looking good*
-[x] bullet 1

**Review by anna-kondratiuk — COMMENTED** [review]
**please approve**

**Review by anna-kondratiuk — COMMENTED** [review]
@AnnaCurly

**Comment by anna-kondratiuk** [comment]
I coudn't do that

**Pull request closed without merging**